live-examples/js-examples/set-prototype-size.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js">var set1 = new Set();
+var object1 = new Object();
+
+set1.add(42);
+set1.add('forty two');
+set1.add(object1);
```

Comment:
Again it might be worth adding something twice here:
```
set1.add('forty two'); 
```

Reply:
ok